### ISSUE TITLE
Prerequisite: Microsoft Visual C++ Redistributable for Visual Studio 2017

### ISSUE THREAD

**Issue by Enhex** [itchio/itch#1185]
Download can be found at https://www.visualstudio.com/downloads/ under "Other Tools and Frameworks".

**Comment by dragmz** [comment]
Here are the direct download links:

x86:
https://download.microsoft.com/download/4/9/5/495e9ce4-e3a3-4055-8499-999dc2474a3c/vc_redist.x86.exe

x64:
https://download.microsoft.com/download/a/b/2/ab2cc1b4-d275-4d73-8d1a-23733eb27763/vc_redist.x64.exe

Can you please add this as a app manifest prerequisite?

**Comment by attilakrupl** [comment]
There are plenty other conditions set in a .prq file. Is it enough to change only the URL or do the other conditions have to be changed too?

**Comment by fasterthanlime** [comment]
I forgot to reply to this, but vc2017 has been available as a prereq for a while:

```
(master ±) itch ツ butler test-prereqs
• Fetching registry...
✓ No prereqs specified, here are those we know about:
+-------------------+-------+----------------------------------------------------------+---------------+
|       Name        | Arch  |                       Description                        |    Version    |
+-------------------+-------+----------------------------------------------------------+---------------+
| dx-june-2010      |   386 | DirectX End-User Runtime (June 2010)                     | June 2010     |
| firejail-386      |   386 | Firejail                                                 | 0.9.53        |
| firejail-amd64    | amd64 | Firejail                                                 | 0.9.53        |
| net-4.5.2         |   386 | Microsoft .NET Framework 4.5.2                           | 4.5.2         |
| net-4.6           |   386 | Microsoft .NET Framework 4.6                             |           4.6 |
| net-4.6.2         |   386 | Microsoft .NET Framework 4.6.2                           | 4.6.2         |
| net-core-2-x64    | amd64 | .NET Core 2 (x64)                                        | 2.0.6.26212   |
| net-core-2-x86    |   386 | .NET Core 2 (x86)                                        | 2.0.6.26212   |
| openal-1.1        |   386 | OpenAL                                                   |           1.1 |
| vcredist-2010-x64 | amd64 | Microsoft Visual C++ 2010 Redistributable (x64)          | 10.0.30319    |
| vcredist-2010-x86 |   386 | Microsoft Visual C++ 2010 Redistributable (x86)          | 10.0.30319    |
| vcredist-2013-x64 | amd64 | Microsoft Visual C++ 2013 Redistributable (x64)          | 12.0.30501    |
| vcredist-2013-x86 |   386 | Microsoft Visual C++ 2013 Redistributable (x86)          | 12.0.30501    |
| vcredist-2015-x64 | amd64 | Microsoft Visual C++ 2015 Redistributable Update 3 (x64) | 14.0.24212.0  |
| vcredist-2015-x86 |   386 | Microsoft Visual C++ 2015 Redistributable Update 3 (x86) | 14.0.23026.0  |
| vcredist-2017-x64 | amd64 | Microsoft Visual C++ 2017 Redistributable (x64)          | 14.10.25017.0 |
| vcredist-2017-x86 |   386 | Microsoft Visual C++ 2017 Redistributable (x86)          | 14.10.25017.0 |
| xna-4.0           |   386 | Microsoft XNA Framework Redistributable 4.0 Refresh      | 4.0.30901.0   |
+-------------------+-------+----------------------------------------------------------+---------------+
```

@attilakrupl I don't understand your question - if it's related to itch (and not InstallShield), please open another issue. Closing this one, since the original problem was resolved!
